### ISSUE TITLE
How to use Adam optimizer instead of SGD in the example given in the document SVDKL

### ISSUE THREAD

**Issue by irum** [cornellius-gp/gpytorch#1555]
Hi,
I am using the  [example](https://docs.gpytorch.ai/en/v1.2.1/examples/06_PyTorch_NN_Integration_DKL/Deep_Kernel_Learning_DenseNet_CIFAR_Tutorial.html) and whilst using SGD optimzer , my accuracy stuck at 50% on my custom dataset. I would like to use Adam instead but I know from the [link](https://docs.gpytorch.ai/en/stable/examples/04_Variational_and_Approximate_GPs/Natural_Gradient_Descent.html#Difference-#2:-Two-optimizers---one-for-the-variational-parameters;-one-for-the-hyperparameters) that Adam can be used  for hyper-parameters so how can I use it in deep kernel learning example for variational parameters. Is using Adam as follows without including variational parameters, the right way?

`optimizer = torch.optim.Adam([
    {'params': model.parameters()},
    {'params': likelihood.parameters()},
], lr=0.1)`

**Comment by gpleiss** [comment]
Yes that is correct.

**Comment by irum** [comment]
Ok. Thank you.
